extensions/blocks/map/map.php

```diff
@@ -25,7 +25,98 @@
 function jetpack_map_block_load_assets( $attr, $content ) {
 	$api_key = Jetpack_Options::get_option( 'mapbox_api_key' );
 
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		static $map_block_counter = [];
+
+		$id = get_the_ID();
+		if ( ! $map_block_counter[ $id ] ) {
```

Comment:
I'm getting a notice here:

```
Notice: Undefined offset: 20 in wp-content/plugins/jetpack-dev/extensions/blocks/map/map.php on line 32
```

Reply:
You're right. It should be:

```suggestion
		if ( ! isset( $map_block_counter[ $id ] ) ) {
```

Reply:
Good catch. https://github.com/Automattic/jetpack/pull/13405/commits/f7d08cbb8b0b66800c64968f63f9c5f5a237bb7b